test/test_skip_systemd.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require_relative "helper"
+require_relative "helpers/integration"
+
+require "puma/plugin"
+
+class TestSkipSystemd < TestIntegration
+
+  def setup
+    skip_unless :linux
+    skip_unless :unix
+    skip_unless_signal_exist? :TERM
+    skip_if :jruby
+
+    super
+
+    ENV["PUMA_SKIP_SYSTEMD"] = "true"
```

Comment:
This needs to be set only for this test, see the many CI failures

Reply:
Same for `NOTIFY_SOCKET`

Reply:
ah shit, how do I do that? Not to well versed in Ruby testing I'm afraid, I thought this _was_ the definition just for this test😅 And when i run the tests locally with `rake test`, the same amount of tests seem to pass as did on my local machine before I made the changes

Reply:
Ah maybe i un-set it in the teardown method?

Reply:
And if so, perhaps for the ENV['NOTIFY_SOCKET'] as well? Is that even a necessary line considering line 28 below? The situation in test_plugin_systemd_jruby.rb is exactly the same as well

Reply:
Ok I ended up just removing the `ENV[...]` lines altogether, since they seem to be set below in the `env` argument to `cli_server` below, lets see what CI says... sorry i'm not developing on a Linux machine so these tests get skipped for me